bin/tectonics-modeling-L0/session1-aspect.ipynb

```diff
@@ -0,0 +1,2840 @@
+{
```

Comment:
<p>ASPECT uses PDE (Partial differential equation) models with finite elements -&gt; ASPECT solves for PDE (Partial Differential Equation) models with finite elements.</p><p> </p><p>Aspect-&gt;ASPECT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>